srtp_cipher_aead_aes_gcm_test.go

```diff
@@ -0,0 +1,85 @@
+package srtp
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSrtpCipherAedAesGcm(t *testing.T) {
```

Comment:
Failing on CI.

Reply:
I had messed up RTCP, used `srtpCipher` instead of `srtcpCipher` when refactoring :(